src/sprints/sprints.controller.ts

```diff
@@ -56,16 +57,28 @@ export class SprintsController {
     constructor(private readonly sprintsService: SprintsService) {}
 
     // dev and admin purpose
+    @CheckAbilities({ action: Action.Manage, subject: "all" })
     @Get()
     @ApiOperation({
-        summary: "gets all the voyages and sprints details in the database",
+        summary:
+            "[Roles:Admin] gets all the voyages and sprints details in the database",
     })
     @ApiResponse({
         status: HttpStatus.OK,
         description: "successfully gets all voyage and sprints data",
         type: VoyageResponseWithoutMeetings,
         isArray: true,
     })
+    @ApiResponse({
```

Comment:
Duplicate `HttpStatus.UNAUTHORIZED` response on line 72 and line 82. I suggest removing one of them.